osmaxx-symbology/colored map.qgs

```diff
@@ -11716,33 +11716,35 @@ def my_form_open(dialog, layer, feature):
             </layer>
           </symbol>
           <symbol alpha="1" clip_to_extent="1" type="line" name="13">
-            <layer pass="0" class="SimpleLine" locked="1">
+            <layer pass="18" class="SimpleLine" locked="0">
```

Comment:
attributes `pass` and `locked` of didn't really change here. This is the [layer previously on line 11736](https://github.com/geometalab/osmaxx-docs/pull/72/files#diff-5f2b846e1b7de868418b1e450c990454L11736).